docs/Developer Documentation/ResumeController/assets/extension_pending_resumption_handler.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
+
+title ExtensionPendingResumptionHandler
+
+
+interface EventObserver {
+   + on_event(event)
+}
+ 
+interface ExtensionPendingResumptionHandler {
+  + HandleResumptionSubscriptionRequest()
+  + on_event(event)
+  + OnResumptionRevert(event)
```

Comment:
looks like the png file does not have this item showed

Reply:
Regenerate all png files